packages/remote-method-guesser/PKGBUILD

```diff
@@ -44,16 +44,19 @@ package() {
   install -dm 755 "$pkgdir/usr/share/$pkgname"
 
   install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
-  cp -a docs "$pkgdir/usr/share/doc/$pkgname/"
+  mv docs/README.md docs/README-docs.md
+  find docs -type f -exec install -m 644 -t "$pkgdir/usr/share/doc/$pkgname/" {} \;
```

Comment:
```suggestion
  mv docs/README.md docs/README-docs.md
  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" docs/* *.md
```

Reply:
This `install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" docs/* *.md` cannot be used. It returns:
```
install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" docs/* *.md
```
due to the presence of subdirectories in `docs`. It was the reason I used `find`.

Reply:
You're right, [install has no recursive option](https://unix.stackexchange.com/questions/165463/problem-with-install-command-to-copy-a-whole-directory).

Reply:
If there are no more changes to implement, can you approve this PR?

Reply:
Thank you for your work on this!

Unfortunately I don't have time to test the package right now. But anyway my approval is optional, and I'm not the one merging.

Reply:
Don't worry. I tested it.